### PR TITLE
LP-2878 Write unit tests for code related to team member invitation

### PR DESCRIPTION
**Pull request by annas-usman**
[LP-2878](https://philanthropyu.atlassian.net/browse/LP-2878)

***

#### Description:
Unite tests for [LP-2840](https://philanthropyu.atlassian.net/browse/LP-2840) added.

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2910/workflows/workflows_id/jobs/2910/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/tests/constants.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_forms.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_models.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_views.py (100%)
-------------
Total:   97 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/admin_dashboard/admin_settings/tests/test_forms.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        'no_status',
+    ]
+)
+def test_invitation_form_and_extract_form_error(
+    email, business_unit, permission, status, expected_is_valid, expected_error
+):
+    """
+    Test the validity of invitation form with different form values and error returned by extract_form_error
+    """
+    business_line = BusinessLineFactory().id if business_unit else None
```
Comment (asadali145) [inline openedx/adg/admin_dashboard/admin_settings/tests/test_forms.py:65]: Can we use business_line fixture from adg/conftest.py?

File: openedx/adg/admin_dashboard/admin_settings/tests/test_views.py [hunk 2]
```diff
@@ -73,7 +77,96 @@ def test_invitations_list_context_data(
...
 
     assert len(actual_invitations_list) == expected_count
+
+
+@pytest.mark.django_db
+@skip_unless_lms
+@pytest.mark.parametrize(
+    'view, url, template, invitations_key, model_key, permission, error',
+    [
+        (admin_dashboard_views.AdminAccountListView, 'admin_accounts', 'accounts.html', 'admins', 'admins', '2', ''),
```
Comment (asadali145) [inline openedx/adg/admin_dashboard/admin_settings/tests/test_views.py:91]: We should follow same line for this as well as we did for the below ones.

**Review by asadali145 — APPROVED** [review]
Nice work!

**Review by AliAdnanSohail — APPROVED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2913/workflows/workflows_id/jobs/2913/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/tests/constants.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_forms.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_helpers.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_models.py (100%)
openedx/adg/admin_dashboard/admin_settings/tests/test_views.py (100%)
-------------
Total:   93 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Pull request merged**